thentos-adhocracy/src/Thentos/Adhocracy3/Backend/Api/Simple.hs

```diff
@@ -375,6 +378,7 @@ thentosApi actionState = enter (enterAction actionState a3ActionErrorToServantEr
 api :: Client.Manager -> AC.ActionState -> Server Api
 api manager actionState =
        thentosApi actionState
+  :<|> Thentos.Backend.Api.Purescript.api (Just "/home/mf/thentos/thentos-purescript/static/")
```

Comment:
Confusingly, this is fixed in #377.

I suggest we merge #377 into master, then master into #376, then fix this by feeding `api` the `ThentosConfig` from `actionState`?